File: tests/Generator/resource/swift/default/Author.swift
// An simple author element with some description
class Author: Codable {
    var title: String
    var email: String
    var categories: Array<String>
    var locations: Array<Location>
    var origin: Location

    enum CodingKeys: String, CodingKey {
        case title = "title"
        case email = "email"
        case categories = "categories"
        case locations = "locations"
        case origin = "origin"
    }
}

File: tests/Generator/resource/kotlin/default/Author.kt

import com.fasterxml.jackson.annotation.*

/**
 * An simple author element with some description
 */
open class Author {
    @JsonProperty("title") var title: String
    @JsonProperty("email") var email: String? = null
    @JsonProperty("categories") var categories: ArrayList<String>? = null
    @JsonProperty("locations") var locations: ArrayList<Location>? = null
    @JsonProperty("origin") var origin: Location? = null
}

